account_due_list_payment_mode/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Due List Payment Mode
+=============================
+
+This module adds a field to the due list of pending payments:
+
+* the *Payment Mode* of invoices on account move lines.
+
+Installation
+============
+
+This module depends on :
+* account_due_list
+* account_payment_partner
+
+This module is part of the OCA/bank-payment suite.
```

Comment:
Put this sentence next to _account_payment_partner_ module to know which one is.